src/core/core/events.js

```diff
@@ -11,24 +8,6 @@ const events = new EventEmitter()
 //more than once and inside of other functions too
 const getState = () => store.getState()
 const getCaptures = ()=> selectors.confirmedCaptures(getState())
-
-const getCapturesCompatible = () => mapKeys(getCaptures(), key => key + 'Capture')
-
-const subscribe = store.subscribe.bind(store)
-
-const subscribeByWatching = (getState, subscribe) => (selector, changeCallback) => {
-  const watcher = watch(()=>selector(getState()), null, isEqual)
-  subscribe(watcher(changeCallback))
-}
-
-//this function allows to subscribe to a selector and listen for when it changes
-const subscribeToStoreByWatching = subscribeByWatching(getState, subscribe)
-
-subscribeToStoreByWatching(state => state.globals.authenticated, isAuthenticated => {
-  if (isAuthenticated) events.emit('ready')
-})
-
-
-events.getCaptures = getCapturesCompatible
+events.getCaptures = getCaptures
```

Comment:
Out of scope of this PR, but it seems weird that we are adding the `getCaptures` function to an `EventEmitter` object. Do you know why we do this rather than exporting it separately?